app/models/cd_api/defendant.rb

```diff
@@ -4,7 +4,7 @@ module CdApi
   class Defendant < BaseModel
     def linked?
       maat_references = offence_summaries.map { |offence| offence&.laa_application&.reference }
-      maat_references.compact.first.present?
+      (maat_references.compact.first.present? && maat_references.compact.first.first != "Z") ? maat_references.compact.first.present? : false
```

Comment:
Could this be unit tested?

I see that there's no model test for this e.g. app/models/cd_api/defendant_spec.rb

I'm not even sure if this model#linked? method is being used 🤔

Reply:
This model#linked? method is used to identify the endpoint for the linked names in the search search results page so i think its definitely used.

Maybe we need to discuss creating a test spec for the model?